src/index.ts

```diff
@@ -86,7 +103,9 @@ const TimeAgo = memo<TimeAgoProps>(
       date,
       locale = navigator.language,
       formatOptions,
+      allowFuture = false,
       hideSeconds = true,
+      hideSecondsText = [],
```

Comment:
Perhaps it might be better to _immediately_ deconstruct it, and pass the deconstructed values to `useCallback`?

That way `useCallback`s dependencies remain as primitives?

Reply:
sure, that works too